src/emails/templates/example/renderExampleEmail.tsx

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { SubscriberRow } from "knex/types/tables";
+import { getEmailL10n } from "../../getEmailL10n";
+import { renderMjml } from "../../renderMjml";
+import { getLocale } from "../../../app/functions/universal/getLocale";
+import { SubscriberBreach } from "../../../utils/subscriberBreaches";
+
+export type ExampleEmailParams = {
+  subscriber: SubscriberRow;
+  breach: SubscriberBreach;
+};
+
+export function renderExampleEmail(
+  template: string,
+  params: ExampleEmailParams,
+): string {
+  const l10n = getEmailL10n(params.subscriber);
+  const options: Intl.DateTimeFormatOptions = {
+    year: "numeric",
+    month: "long",
+    day: "numeric",
+  };
+  const dateFormatter = new Intl.DateTimeFormat(getLocale(l10n), options);
+
+  return renderMjml(template, {
+    "{data-breach-header-1}": l10n.getString("data-breach-header-1"),
+    "{data-breach-body-1}": l10n.getString("data-breach-body-1", {
```

Comment:
I find using the full message id in the template rather odd, as it suggests to a reader that they might _need_ to be the same, but that's just a name that's local to the template and this function.

Reply:
Good point, that can be confusing. Renamed in 757251a86.